openLoopDepth: true
openLoopDepthSpeed: -6.0 #only used when openLoopDepth = true
closeLoopDepth: 0.5      #only used when openLoopDepth = false

surgeSpeed: -2.0
